tests/performance/locust_settings/_dump_dotenv.py

```diff
@@ -0,0 +1,13 @@
+import json
+from typing import TypeVar
+
+from pydantic_settings import BaseSettings
+
+T = TypeVar
+
+
+def dump_dotenv(settings: BaseSettings):
+    result = [
+        f"{key}={val}" for key, val in json.loads(settings.model_dump_json()).items()
```

Comment:
would not `settings.model_dump()` do the same as `json.loads(settings.model_dump_json())`

Reply:
No, because the `timedelta` object is serialized into the format which `locust` expects. I.e. `0:10:20` is serialized to `0h10m20s` so that locust can understand it.

Reply:
@bisgaard-itis FYI you also have control on the serialization encoding . For another time then